modules/class-simplechart-request-handler.php

```diff
@@ -85,13 +85,17 @@ public function frame_id() {
 	 * @return none
 	 */
 	private function _handle_api_request( $id ) {
+		// Allows same-origin https CORS requests
+		header( 'Access-Control-Allow-Origin: *' );
```

Comment:
I think it's ok because the only possible outcome if you get this far is sending some JSON then exiting, there's no possibility of writing anything to the database or file system, etc